models/engine/db_storage.py

```diff
@@ -74,3 +74,29 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+
+    def get(self, cls, id):
+        """A method to retrieve one object"""
+        if cls not in classes.values():
+            return None
+        all_values = models.storage.all(cls)
+        for value in all_values().values():
+            if value.id == id:
+                return value
+        return None
+
+    def count(self, cls=None):
+        """A method to count the number of objects in storage"""
+        all_classes = classes.values()
+        if not cls:
+            i = 0
+            for c in all_classes:
```

Comment:
for variables i and c, maybe you can be more clear and specify better whar is the variable for.